mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -117,18 +117,19 @@ public void add(MailboxSession session, Mailbox mailbox, MailboxMessage message)
             indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJson(message, ImmutableList.of(session.getUser())));
         } catch (Exception e) {
             try {
-                LOGGER.warn("Indexing mailbox {}-{} of user {} on message {} without attachments ",
+                LOGGER.warn(String.format("Indexing mailbox %s-%s of user %s on message %s without attachments ",
```

Comment:
Are you sure, have you tested that this is a issue ?
https://stackoverflow.com/questions/5951209/how-to-log-exception-and-message-with-placeholders-with-slf4j.
And when I was working at AF I had the remember this did work.
We use 1.6.6 so it should be ok.